src/pages/Environments.js

```diff
@@ -108,6 +150,12 @@ const Environments = () => {
     if (!_.some({ id: getDeleteDiskId() }, newDisks)) {
       setDeleteDiskId(undefined)
     }
+    if (!_.some({ appName: getErrorAppId() }, newApps)) {
```

Comment:
I agree that fixing all unnecessary usage is beyond the scope of this PR, but let's not add more. I've added just a few suggestions to help out.
```suggestion
    if (!_.some({ appName: errorAppId }, newApps)) {
```

Reply:
made updates to not use getters anymore, thanks for pointing that out!